phoenix/slackbot/tasks.py

```diff
@@ -533,8 +519,10 @@ def add_comment(message_ts, channel_id, comment, icon_url=None, username=None):
 @shared_task
 def notify_users():
     now = arrow.utcnow().shift(minutes=settings.NOTIFY_BEFORE_ETA)
-    for outage in Outage.objects.filter(resolved=False).exclude(eta=0):
-        # notify only if eta is known (eta=0 => unknown)
+    for outage in Outage.objects.filter(resolved=False):
+        eta_deadline = outage.eta_deadline
```

Comment:
Hm, could we exclude it in the query?

Reply:
It could make things complicated in the future. Right now the `eta_deadline` will be `False` if the `eta` value includes symbol `>`. This whole logic is hidden behind methods (properties) of `Outage` and evaluated  according to that specific logic. It would be required to basically duplicate this logic and when the values change this would have to be modified in two places.

Also I think there's high chance of introducing bug if this logic changes as this is separate cron task and it might be overlooked.

As to performance, having multiple open `resolved=False` outages is very unusual so there isn't a performance gain.

Reply:
Aha, I see, thought that it is a DB field, but it is actually a value calculated on the python level. ok, lets leave it as is